documentation/src/main/asciidoc/ch11.asciidoc

```diff
@@ -160,8 +160,8 @@ javax.faces.validator.BeanValidator.MESSAGE={0}
 [[section-integration-with-cdi]]
 === CDI
 
-As of version 1.1, Bean Validation is integrated with CDI (Contexts and Dependency Injection for
-Java^TM^ EE).
+As of version 1.1, Jakarta Bean Validation is integrated with CDI (Contexts and Dependency Injection for
```

Comment:
this does sound a bit strange :smiley: as there won't be a Jakarta BV 1.1 or did I miss something ? Maybe instead we should just say that `Jakarta Bean Validation is integrated with CDI` ?

Reply:
I had an hesitation about this one when I changed it.

I made it `As of version 1.1, Bean Validation (and therefore Jakarta Bean Validation)`.